devicetypes/smartthings/bose-soundtouch.src/bose-soundtouch.groovy

```diff
@@ -28,6 +28,8 @@ metadata {
         capability "Refresh"
         capability "Music Player"
         capability "Health Check"
+		capability "Sensor"
```

Comment:
can you fix the indentation for this one for now? we'll make an update to all the files later to move to tabs or spaces